test/parallel/test-runner-coverage-source-map.js

```diff
@@ -40,6 +41,28 @@ describe('Coverage with source maps', async () => {
     const spawned = await common.spawnPromisified(process.execPath, flags, {
       cwd: fixtures.path('test-runner', 'coverage')
     });
+
+    t.assert.strictEqual(spawned.stderr, '');
+    t.assert.ok(spawned.stdout.includes(report));
+    t.assert.strictEqual(spawned.code, 1);
+  });
+
+  await it('should only work with --enable-source-maps', async (t) => {
```

Comment:
You never need to `await` a test in a suite because the suite does it for you. I realize that you're being consistent with the rest of the file, so it's more of an FYI.